tests/test-run.sh

```diff
@@ -113,6 +113,15 @@ $RUN --unshare-pid --as-pid-1 --bind / / bash -c 'echo $$' > as_pid_1.txt
 assert_file_has_content as_pid_1.txt "1"
 echo "ok - can run as pid 1"
 
+if ! test -u ${BWRAP}; then
+    echo "ok - # SKIP no --cap-add support"
+else
+    $BWRAP --unshare-all --uid 0 --gid 0 --cap-add ALL --bind / / --proc /proc \
```

Comment:
One issue with all of our tests is we're using `--bind / /` which IMO is an unusual/weird case.   Also you seem to be skipping these tests in the non-setuid case - why?
 
I pushed a fixup for both of these.

Reply:
we are skipping them in the non-setuid case as bwrap will complain that only root can use `cap-add`

Reply:
are you fine if we add again the check for non-setuid case?

Reply:
It's not about setuid but about support for user namespaces....which is currently sort of the same thing because if the distro has user namespaces, there's no reason to install bwrap setuid.

To say this another way, it's fine to use `--cap-add ALL` as non-root if `--unshare-user`, right?

On Fedora we definitely shouldn't be installing setuid, and the tests definitely work for me currently.  We may however need the suid check on CentOS7 today.

Reply:
Found the bug where we were mistakenly doing `chmod u+s` on Fedora.